### ISSUE TITLE
Withdrawal

### ISSUE THREAD

**Issue by clapoujade** [eric-taix/volley34-mobile#214]
It is not possible to enter withdrawal in the score board. Or it is not very intuitive

**Comment by eric-taix** [comment]
Good point. Currently there's no way to enter a withdrawal. Usually I think that withdrawal is "decided" by Laurent because either a team has been registered in a competition but cannot play (no matter why) or either the withdrawal to a team which can not play a postponed match. But I'm not sure that a team  wants to withdraw from a match voluntarily.

Anyway I move it to Release V2 for discussion

**Comment by clapoujade** [comment]
That's fine then.
Perhaps a tip that explain the alternate procedure (By mail or whatever) .... because the user could be lost in such case. 
.
